### ISSUE TITLE
scm-server.service exits with JAVA_HOME and execution error

### ISSUE THREAD

**Issue by mshortall** [scm-manager/scm-manager#1195]
## Issue description

After installing scm-server-2.0.0 on a CentOS 6.9 VM, the systemctl start scm-server.service exited and failed to start. The logging indicates that JAVA_HOME is not set, and that /usr/lib/jvm/jre-11/bin/java could not be executed.

Java 1.8.0 is installed, however setting JAVA_HOME did not resolve the issues with starting scm-manager. Installing Java 11 resolved the issue.

#### Describe how to reproduce the bug

Install CentOS 6.9
Install java-1.8.0-openjdk
install scm-server-2.0.0

The scm-server rpm installation attempts to start scm manager and renders an error with exit status 1. journalctl -xe will provide details about the JAVA error.

#### Describe your SCM-Manager setup

What plugins and what versions do you use?
scm-git-plugin
scm-hg-plugin
scm-legacy-plugin
scm-svn-plugin

Are you using a server like tomcat?
nginx

Do you have special settings like proxies? Are you running your instance behind a reverse proxy?
behind reverse proxy

#### Which operating system, browser and versions do you use?
CentOS Release 6.9
Google Chrome V 83.0.4103.97

Windows, Mac or Linux? Chrome, Firefox or IE? What versions?
MAC & Linux
Google Chrome V 83.0.4103.97

#### Log of the bug

Jun 16 20:45:50 scm.example.com scm-server[27549]: Error: JAVA_HOME is not defined correctly.
Jun 16 20:45:50 scm.example.com scm-server[27549]: We cannot execute /usr/lib/jvm/jre-11/bin/java
Jun 16 20:45:50 scm.example.com systemd[1]: scm-server.service: control process exited, code=exited status=1

#### Further information like screenshots

Installing java-11-openjdk resolved the issue with starting SCM with systemctl start scm-server.service

**Comment by sdorra** [comment]
How did you set `JAVA_HOME`? Have you used the `/etc/default/scm-server` file?

**Comment by mshortall** [comment]
JAVA_HOME was not set on the server while running v1.6.0. After seeing the above error logged, I did set JAVA_HOME to /usr/lib/jvm/jre while installing the java-11-openjdk package.

/etc/default/scm-server is in place, but the JAVA_HOME line is commented out while running v1.6.0. I was not familiar with this file, or I would have tried uncommenting that line after the migration to v2.0.0 and restarting scm. Instead I just exported /usr/lib/jvm/jre to /etc/profile to define JAVA_HOME.
